glhe/topology/ground_heat_exchanger_long_time_step.py

```diff
@@ -113,7 +113,7 @@ def simulate_time_step(self, inputs: SimulationResponse) -> SimulationResponse:
 
         c_2 = (self.c_0 * g + resist_b * g_b)
 
-        cp = self.fluid.get_cp(inlet_temp)
+        cp = self.fluid.cp(inlet_temp)
```

Comment:
Operate directly on the scp class instances, rather than having an intermediate class and function.